jsonschema-core.xml

```diff
@@ -2757,9 +2756,9 @@
                     Implementations MAY elect to provide additional information.
                 </t>
 
-                <section title="Keyword Relative Location">
+                <section title="Subschema Relative Location">
```

Comment:
I think it would make more sense to talk about the dynamic scope location, or evaluation path, or something.  For one thing, this includes root schemas as well as subschemas.  And for another, as I've been digging into keyword behaviors, I've really wanted to be more clear and consistent on the difference between a dynamic scope (the application of a schema to an instance, identified by `evaluationPath` + `instanceLocation`) and a schema (identified by a URI).

This is the distinction that the whole "relative location" vs "absolute location" was trying to get at, and talking about dynamic scopes vs schema locations is much more accurate.